styles/abstracts/_theme.scss

```diff
@@ -51,7 +51,7 @@ $themes: (
     'k-accentlight': #ffe5f3,
     'k-accentHover': #363234,
     'k-accentlight2': #363234,
-    'k-accentlight2-dark': #ccc,
+    'k-accentlight2-dark': #ffffff,
```

Comment:
This does not seem right cc @exezbcz

Reply:
`#363234` - this is accentlight2-dark

Reply:
> `#363234` - this is accentlight2-dark

that is also `k-accentlight2`

so maybe is there some diff?